src/main/webapp/resources/js/pages/projects/samples-metadata-import2/components/SampleMetadataImportMapHeaders.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { navigate } from "@reach/router"
+import {
+  Button,
+  Radio,
+  Typography,
+} from "antd";
+import { SampleMetadataImportWizard } from "./SampleMetadataImportWizard";
+import { useSelector } from "react-redux";
+import { BlockRadioInput } from "../../../../components/ant.design/forms/BlockRadioInput";
+
+const { Text } = Typography
+
+function Back() {
+  navigate(-1);
+}
+
+/**
+ * React component that displays Step #2 of the Sample Metadata Uploader.
+ * This page is where the user selects the sample name column.
+ * @returns {*}
+ * @constructor
+ */
+export function SampleMetadataImportMapHeaders({ projectId }) {
+
+  const { headers, sampleNameColumn } = useSelector((state) => state.reducer);
```

Comment:
When I tried this, there was not a selected column.  If a default column cannot be selected, just select the first option by default since that is usually where identifiers are on excel files.

Reply:
Even with a valid sample name in this column, nothing was selected

Reply:
Was the sample within the current project??? Please see 41c8b8f7ecc1f3ddbe91969526ec293762a1b9d3.

Reply:
Please see 41c8b8f7ecc1f3ddbe91969526ec293762a1b9d3. Was the sample within the project?

Reply:
As discussed, we are going to look through every row until a sample is found. Please see 5a6597abad7c0cebf92cbc0cab9c672360d9d3c0 and 068b5dc709fc7431dbdfe1b48b58961f44917f67.